frontend/packages/integration-tests-cypress/tests/app/demo-dynamic-plugin.spec.ts

```diff
@@ -13,13 +13,15 @@ const PLUGIN_PULL_SPEC = Cypress.env('PLUGIN_PULL_SPEC');
 /* The update wait is the value to wait for the poll of /api/check-updates to return with the updated list of plugins
  after the plugin is enabled and loaded. This wait will be longer on ci than when debugging locally. */
 const CHECK_UPDATE_WAIT = 300000;
+const WAIT_OPTIONS = { requestTimeout: CHECK_UPDATE_WAIT };
```

Comment:
nit: I'd inline this since it's only used once

Reply:
Thanks, @spadgett.  I had a version that was reusing this and forgot to clean up.